Source/capture.cpp

```diff
@@ -90,18 +90,16 @@ BOOL __fastcall CapturePix(HANDLE hFile, WORD width, WORD height, WORD stride, B
 	BYTE *pBuffer, *pBufferEnd;
 
 	pBuffer = (BYTE *)DiabloAllocPtr(2 * width);
-	do {
-		if (!height) {
-			mem_free_dbg(pBuffer);
-			return TRUE;
-		}
-		height--;
+	while (height-- != 0) {

```

Comment:
Not a big fan of the way they are doing things :) I guess technically height-- could go inside the loop, but This also seems likely. We do have a few place where it's in side the loop, maybe we should move them to the while part.

Reply:
I know. First I went with `for (i = 0; i < height; i++)` and other variations. This one was the one i got bin perfect.

Reply:
Could you do:
```
while(height != 0) {
  height--;
```
???